### PR TITLE
coin-utils: allow cross-build + fix random install issues

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

Cross-build just works (I've tested Macos M1 and iOS with apple-clang 12).

And fixes https://github.com/conan-io/conan-center-index/issues/5820

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by uilianries — DISMISSED** [review]

**Comment by SpaceIm** [comment]
hum, shared lib is empty while cross-building. static lib is fine.

**Comment by SpaceIm** [comment]
I guess that issue comes from libtool, but we can't call autoreconf, coin-utils m4 macros are missing in tarball...

**Review by prince-chrismc — APPROVED** [review]

File: recipes/coin-utils/all/conanfile.py [hunk 1]
```diff
@@ -128,7 +128,7 @@ def package(self):
         self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
         with self._build_context():
             autotools = self._configure_autotools()
-            autotools.install()
+            autotools.install(args=["-j1"])
```
Comment (prince-chrismc) [inline recipes/coin-utils/all/conanfile.py:131]: Is this required? maybe add a comment if it is?

Comment (SpaceIm) [reply recipes/coin-utils/all/conanfile.py:131]: it is
https://github.com/conan-io/conan-center-index/issues/5820

Comment (SSE4) [reply recipes/coin-utils/all/conanfile.py:131]: could be put under the `if macos` condition, but install is usually fast

**Review by SSE4 — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Pull request merged**